lib/numo/sparse/csc.rb

```diff
@@ -47,6 +66,7 @@ def nnz
         end
         @data = narray.class[*data]
         @indices = Numo::Int32[*indices]
+        @indices_temp = Numo::Int32[*indices_temp]
```

Comment:
I don't like this approach.
Transposed indices should be created in `transpose` method on demand.